pkgs/desktops/gnome/extensions/extensionRenames.nix

```diff
@@ -26,4 +26,16 @@
 
   # That extension is broken because of https://github.com/NixOS/nixpkgs/issues/118612
   #"flypie@schneegans.github.com" = null;
+
+  # These are overrides for the manually packaged ones (some of which got deleted – as in: replaced with
+  # some automatic packaging –, but we still need the "old" name for backwards compatibility).
```

Comment:
Maybe something like: “These are overrides for extensions that were manually packaged in the past but were replaced by automatic packaging. We preserve the old attribute name for backwards compatibility.”

Reply:
Though maybe we could just create aliases and switch to the new names (where they do not contain typos).

Reply:
I just added the extension slug as a comment to all of them. Most of the renames are really sensible and I'd like to keep them. A few could indeed be replaced by an alias, you decide if it should be done in this PR (or if it is even worth the effort).

Reply:
The switch to aliases does not need to be done in this PR. Just something to keep in mind if we decide that some of the automatic names are more sensible than the old manual ones.